migrator/migrations/m_173_to_m_174_create_notification_schedule_table/migration.go

```diff
@@ -0,0 +1,37 @@
+package m173tom174
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	frozenSchema "github.com/stackrox/rox/migrator/migrations/frozenschema/v75"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"gorm.io/gorm"
+)
+
+var (
+	startSeqNum = 173
+	migration   = types.Migration{
+		StartingSeqNum: startSeqNum,
+		VersionAfter:   &storage.Version{SeqNum: int32(startSeqNum + 1)},
+		Run: func(database *types.Databases) error {
+			// Migration code comes here
```

Comment:
```suggestion

```

Reply:
Moved to https://github.com/stackrox/stackrox/pull/5067